src/reducers/linter.tsx

```diff
@@ -177,7 +177,12 @@ export const fetchLinterMessages = ({
   url: string;
   versionId: number;
 }): ThunkActionCreator => {
-  return async (dispatch) => {
+  return async (dispatch, getState) => {
+    // See: https://github.com/mozilla/addons-code-manager/issues/591
+    if (getState().linter.isLoading) {
```

Comment:
Could you also log a warning here? It could be surprising that nothing is happening when the action is dispatched.

Reply:
yep, although I'll rather log a `debug` message.